src/app/_shared/services/list-view/service-list-view.component.html

```diff
@@ -1,9 +1,30 @@
-<!--TODO(FJMentz): Do date stuff (commented)-->
-
 <div id="serviceList">
     <div *ngFor="let service of services; let i = index">
         <div class="card ServiceListCard">
             <div class="card-header" [attr.id]="'heading-'+ i" data-toggle="collapse" [attr.data-target]="'#collapse-' + i" aria-expanded="true" [attr.aria-controls]="'collapse-' + i">
+<<<<<<< HEAD
```

Comment:
Merge conflict here

Reply:
haha saw it just now, pushing as we speak